lib/api/renderers.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework_csv.renderers import CSVRenderer
+
+class CSVExcelRenderer(CSVRenderer):
+    format = 'excel.csv'
+    writer_opts = { 'delimiter': ';' }
+
+    def remove_newlines(self, x):
```

Comment:
no need to be member, can be a function.

Reply:
Should also make sure that v is a string, if that is not the case always.

Reply:
We check if it's a string or not in `remove_newlines`, which acts as an identity function if not `isinstance(x, str)`